package.json

```diff
@@ -151,7 +151,7 @@
     "redux-logger": "^2.0.2",
     "redux-thunk": "^0.1.0",
     "semver-regex": "^1.0.0",
-    "sequelize": "^4.37.0",
+    "sequelize": "^4.37.10",
```

Comment:
You shouldn't need to change this. The "^" means npm will use the latest version of 4.x.x. You generally only need to update it when upgrading the major version or adding a new package.

And for future reference: when you do need to change it here, you would also need to run npm install and check in the updated package-lock.json. (The build is failing right now because package.json doesn't match package-lock.json.)